Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -529,8 +529,9 @@ void OnClickPerformed(InputAction.CallbackContext ctx, EventSource eventSource,
 
         void OnScrollWheelPerformed(InputAction.CallbackContext ctx)
         {
-            var scrollDelta = ctx.ReadValue<Vector2>();
-            if (scrollDelta.sqrMagnitude < k_SmallestReportedMovementSqrDist)
+            // ISXB-704: convert input value to uniform ticks before sending them to UI.
+            var scrollTicks = ctx.ReadValue<Vector2>() / InputSystem.scrollWheelDeltaPerTick;
```

Comment:
Is InputSystem.scrollWheelDeltaPerTick guaranteed to never be zero (to make sure we never can run into division by zero)

Reply:
If trunk version guarantees that its never zero I believe its fine to not check it in package. Agree that its better to define the API contract on lower level.